chef_master/source/resource_windows_ad_join.rst

```diff
@@ -60,6 +61,11 @@ Properties
 
    Required. The domain user that will be used to join the domain.
 
+``new_hostname``
+  **Ruby Type:** String
+
+  "Specifies a new name for the computer in the new domain."
+
```

Comment:
On a new line here you'll want to say "New in Chef Client 14.5."

Reply:
Other examples I found had it at the end, let me know if it should be changed.